.travis.yml

```diff
@@ -1,2 +1,13 @@
 language: node_js
 script: make test
+before_deploy:
+- npm-prepublish --verbose
+deploy:
+  provider: npm
+  email: "replace-with-email"
+  api_key:
+    secure: "replace-with-encrypted-api-key"
```

Comment:
I think it makes sense for someone who is a collaborator to do this @dgraham would you mind?